CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
-## 4.1.1
+## 4.1.2
+- Remove ES 1.x from test matrix. The testing requirements were too severe, and update bugs were too troublesome to correct. See more about the update bugs @ https://github.com/elastic/elasticsearch/issues/19103
```

Comment:
Maybe indicate something saying how long ES 1.x hasn't worked for this plugin, just in case someone interprets the 4.1.1 version as "Supports ES 1.x" -- I'd like to avoid some confusion :)